### ISSUE TITLE
[bug] can not get the echarts instance

### ISSUE THREAD

**Issue by xialvjun** [ecomfe/vue-echarts#542]
> ## Help / 帮助
>
> Replace the `[ ]` with `[x]` to check an option. / 将 `[ ]` 替换为 `[x]` 以选择对应选项。

## The type of this issue / Issue 类型

- [ ] Feature request / 新特性需求
- [x] Bug report / Bug 报告

## Not introduced by ECharts / 非 ECharts 本身问题

Problems about ECharts itself are not handled in this repo. / 本 repo 不负责处理 ECharts 本身的问题。

- [x] I've checked it's not a problem of ECharts itself. / 我已检查过，这个问题非 ECharts 本身的问题。

### Vue version / Vue 版本

- [ ] Vue 3
- [x] Vue 2

### How are you importing Vue-ECharts? / 你是如何引入 Vue-ECharts 的？

- [x] Importing `vue-echarts` with a bundler environment / 在 webpack 等打包工具环境下引入 `vue-echarts`
- [ ] Using the global variable by including `<script>` tags / 通过 `<script>` 标签引入全局变量

### The version of Vue-ECharts you are using / Vue-ECharts 的版本

> 6.0.0-beta.6

## Reproduction link / 复现链接

[Vue 2](https://codesandbox.io/s/compassionate-kalam-t299z?file=/src/App.vue) 无法通过 `this.$refs.chart.chart` 获取 `echarts instance`, 明明看 `vue-echarts` 源代码里有这句代码, 应该是有暴露出来的啊, 但是 `mounted` 里 `console.log` 得到的是 `undefined`
```ts
Object.defineProperty(exposed, "chart", {
    get: function () {
        return unref(chart);
    }
});
```
怀疑是 vue2 对 setup 函数返回的对象做解构，丢失了 getter 属性。也许把 exposed 换成 getCurrentInstance 就可以

**Comment by Moxie1776** [comment]
This is what is working for me that I have triggered with a dataZoom event.

```
    eventChartZoomScada() {
      const data = this.$refs.chartScada;
      const startDate = new Date(data.getOption().dataZoom[0].startValue);
      const endDate = new Date(data.getOption().dataZoom[0].endValue);
    },
```

**Comment by xialvjun** [comment]
@Moxie1776 It seems you were trying to reply to another issue.

**Comment by Justineo** [comment]
Fixed in `6.0.0-rc.5`.
